recipes/tides-ml/meta.yaml

```diff
@@ -24,12 +24,12 @@ requirements:
     - pip
   run:
     - python <=3.11.7
```

Comment:
Python <=3.11.7? That's peculiar, is Python 3.11.8 really breaking something for you?